examples/application_commands/slash_command_trees.py

```diff
@@ -0,0 +1,44 @@
+import discord
+from discord import app_commands
+
+client = discord.Client()
+
+# initiate our command tree, with the client
+apple_tree = app_commands.CommandTree(client)
```

Comment:
Consider renaming this and calling it just `tree`. The novelty is nice, but for an example it could be confusing to the first time user.